hips/tiles/tile.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from .description import HipsDescription
+import numpy as np
+
+__all__ = [
+    'HipsTile',
+]
+
+
+class HipsTile:
+    """HiPS tile container.
+
```

Comment:
Document `Parameters` here, and also add an `Examples` section, showing how to fetch a tile and how to access it's pixel data.